### PR TITLE
Corrected some Grammar, Spelling and changed some wording.

### PR DESCRIPTION
**Pull request by Difluoride**

### CODE REVIEW

File: spawners/soulshards.md [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+| 2    | 128                 | 4      | 4     | X                   | X                |
+| 3    | 256                 | 5      | 3     | X                   | X                |
+| 4    | 512                 | 5      | 3     | ✓                   | X                |
+| 5    | 1024                | 10     | 2     | ✓                   | ✓                |
 
-## [Soul Cage](@)
-The second main component of this plugin is the "Soul Cage", this is where you can put your soul shard and make a spawner!
-You can right click this if you ever want a bit of info about what this spawner currently is.
+## [Obtaining a Soul Cage](@)
+The second core component of this plugin is the "Soul Cage". This item is the only way you can put your Soul Shard to use, and make your own spawner!
```
Comment (djkittycoolcat) [inline spawners/soulshards.md]: I would add that a Soul Cage is like a Spawner. It's a little ambiguous here...

**Comment by Difluoride** [comment]
Goddamn approve this!

**Comment by WalshyDev** [comment]
Please do these other changes as part of a different branch

**Comment by Difluoride** [comment]
_**Sigh**_ Yup, no worries. Will have to be done later

**Pull request merged**